JS/Lesson22/onder/script.js

```diff
@@ -0,0 +1,8 @@
+let acc = document.getElementsByClassName("accordion");
+
+for (let i = 0; i < acc.length; i++) {
+    acc[i].onclick = function () {
+        let panel = this.nextElementSibling;
```

Comment:
Since the value of this variable is never change it's better to declare it using `const` keyword:
```suggestion
        const panel = this.nextElementSibling;
```